docs/guides/app-payments/build-on-mobile.mdx

```diff
@@ -205,15 +205,17 @@ class AppDelegate: NSObject, UIApplicationDelegate {
 </Tabs>
 
 :::tip Experimental Feature for a pilot period.
```

Comment:
Remove everything about this experimental feature. Decision was made to drop it completely.